RoboFile.php

```diff
@@ -113,36 +176,35 @@ public function testFull(int $drupal_version = 9, string $site_name = NULL) {
 
     // Test all the Solr things.
     $this->testSolrEnabled($site_name);
-
-    // Test creating Solr index.
-    $this->testSolrIndexCreate($site_name, 'dev');
-
-    // Test running the reload
-    $this->testPantheonSolrReload($site_name, 'dev');
```

Comment:
Are you on purpose dropping these Solr and pantheon solr reload tests? I'm unsure what they entail but looks like some functionality we probably want to have tests on them unless I'm missing something

Reply:
Looks like `testPantheonSolrReload ` has been removed entirely: https://github.com/pantheon-systems/search_api_pantheon/pull/203/files/a111b2fc51e923266d221373e014f1128e250012#diff-fed7dd361d1a8d696657705806d8e756ff49cc4659f2191ab0e91172a59be0d8L812

and  `testSolrIndexCreate` was mostly rewritten: https://github.com/pantheon-systems/search_api_pantheon/pull/203/files/a111b2fc51e923266d221373e014f1128e250012#diff-fed7dd361d1a8d696657705806d8e756ff49cc4659f2191ab0e91172a59be0d8L782-L839

Reply:
I will add them back after I'm sure they're needed. Right now, I just want to get the D11 release in the wild.